command/ca/renew.go

```diff
@@ -558,6 +576,44 @@ func (r *renewer) Daemon(outFile string, next, expiresIn, renewPeriod time.Durat
 	}
 }
 
+// RenewAfterExpiry creates an authorization token with the given certificate
+// and attempts to renew the expired certificate.
+func (r *renewer) RenewAfterExpiry(cert tls.Certificate) (*api.SignResponse, error) {
+	var issuer string
+	if ext, ok := provisioner.GetProvisionerExtension(cert.Leaf); ok {
```

Comment:
how does this work for certificates authenticated by RA but signed by a different CA (e.g., Vault RA)?

Reply:
Renewal is not yet supported on RA mode, but ef532b6 changes this to use a fixed string.